okhttp/src/test/java/io/grpc/transport/okhttp/OkHttpClientTransportTest.java

```diff
@@ -478,7 +478,7 @@ private static Buffer createMessageFrame(byte[] message) {
   private static class MockFrameReader implements FrameReader {
     CountDownLatch closed = new CountDownLatch(1);
     boolean throwExceptionForNextFrame;
-    boolean nextFrameResult = true;
+    Boolean nextFrameResult = null;
```

Comment:
Could we just instead call this "returnFalseInNextFrame" and it only becomes true?